admin/src/pages/SettingsPage/index.tsx

```diff
@@ -441,6 +448,21 @@ const SettingsPage = () => {
                             />
                           </GridItem>
                         )}
+                        <GridItem col={4} s={12} xs={12}>
```

Comment:
I think this input could be right next to the allowed levels' input since they're related.

Reply:
> Could even be before the allowed levels since this one overwrites it.

Not quite sure, but i think it does not override the allowed levels option. It literally just removes the disabled on the field and disables the payload transformation for attached cascade. Which is fine, think its important to cap menu attached levels as option.